src/sssom/cliques.py

```diff
@@ -27,9 +27,13 @@
 from .sssom_document import MappingSetDocument
 from .util import MappingSetDataFrame
 
+if TYPE_CHECKING:
+    import networkx
 
-def to_digraph(msdf: MappingSetDataFrame) -> nx.DiGraph:
+def to_digraph(msdf: MappingSetDataFrame) -> "networkx.DiGraph":
```

Comment:
We really do not need this method at all, but its a good first step to remove the networkx dependency

Reply:
I'm not against having a first-party export function to convert SSSOM mappings into a graph I think this is valuable, but having it mixed up with the cliques.py code is not great